src/integration-test/java/org/ow2/proactive/catalog/rest/controller/BucketControllerIntegrationTest.java

```diff
@@ -331,7 +331,7 @@ public void testDeleteNonEmptyBucket() throws IOException {
                .queryParam("kind", "myobjectkind")
                .queryParam("name", "myTestName")
                .queryParam("commitMessage", "first commit")
-               .queryParam("contentType", "application/xml")
+               .queryParam("objectContentType", "application/xml")
```

Comment:
Can't you use `MediaType.APPLICATION_XML.toString()` here as well?

Reply:
Indeed I think that MediaType. APPLICATION_XML_VALUE would be better. I'll change that.